### ISSUE TITLE
Visual ATM Model Report

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#1105]
**Describe the solution you'd like**

The [Auto Tune Models (ATM)](https://github.com/HDI-Project/ATM) Python library is an easy-to-use classification model solver that searches for the best model given a CSV dataset containing features and a target. During its run it creates a SQLite database that stores results from its auto-tuning and can be accessed using a results object with summary data, scores, and the best model found. ATM also has a CLI and a REST API. 

To take advantage of ATM, a Yellowbrick `contrib` module (e.g. `yellowbrick.contrib.atm`) should implement visual diagnostics functionality for the ATM results object, allowing the user to explore classification visualizers for the best classifier, or compare classification visualizers across multiple models. Note that ATM may be an excellent start to getting multi-model report functionality from Yellowbrick, since ATM wraps a suite of trained and cross-validated models. 

Open questions include:

- Should Yellowbrick directly access ATM's database?
- What data does ATM provide that could enable ATM-specific visualizations?
- Can Yellowbrick be used with the REST API?

A successful conclusion to this issue is the creation of an `yellowbrick.contrib.atm` package with the following functionality:

- [ ] A wrapper for `atm.Model` and/or `atm.Datarun` that enables Yellowbrick classifier visualizers 
- [ ] Documentation/blog post about how to integrate Yellowbrick and ATM
- [ ] Follow on issues for ATM-specific visualizers and functionality 

**Is your feature request related to a problem? Please describe.**

This PR is related to #397 that described using Yellowbrick with other ML libraries. Since this discussion, Yellowbrick has incorporated contrib support for 3rd party libraries using wrappers and other methods (see #1103) and has been used successfully with [other projects like Keras](https://towardsdatascience.com/evaluating-keras-neural-network-performance-using-yellowbrick-visualizations-ad65543f3174). The ATM library, however, will not work with the wrapper since it uses sklearn under the hood and has extended functionality that Yellowbrick could take advantage of, such as multi-model comparisons. Because of this, an ATM contrib model would be well suited to Yellowbrick's out of core approach. 

**Examples**

N/A

@mitevpi any thoughts on this?
